admin/app/controllers/authenticated/organizations/get.js

```diff
@@ -57,17 +61,51 @@ export default class GetController extends Controller {
     return this.store.createRecord('membership', { organization, user })
       .save()
       .then(async () => {
-        this.userEmail = null;
+        this.userEmailToAdd = null;
         this.notifications.success('Accès attribué avec succès.');
       })
       .catch(() => {
         this.notifications.error('Une erreur est survenue.');
       });
   }
 
+  @action
+  createOrganizationInvitation() {
+    this.isLoading = true;
+    if (!this._isEmailToInviteValid(this.userEmailToInvite)) {
+      this.isLoading = false;
+      return;
+    }
+    const email = this.userEmailToInvite.trim();
+
+    return this.store.createRecord('organization-invitation', { email })
+      .save({ adapterOptions: { organizationId: this.model.id } })
+      .then((organizationInvitation) => {
+        this.notifications.success(`Un email a bien a été envoyé à l'adresse ${organizationInvitation.email}.`);
+        this.userEmailToInvite = null;
+      })
+      .catch(() => this.notifications.error('Une erreur s’est produite, veuillez réessayer.'))
+      .finally(() => this.isLoading = false);
+  }
+
   _toArrayWithUnique(targetProfilesToAttach) {
     const trimedTargetProfilesToAttach = targetProfilesToAttach.split(',').map((targetProfileId) => targetProfileId.trim());
     return _.uniq(trimedTargetProfilesToAttach);
   }
 
+  _isEmailToInviteValid(email) {
```

Comment:
```suggestion
  _isEmailValid(email) {
```
Ca pourrait marcher même pour d'autres emails si je ne me trompe pas.

Reply:
Les règles pourraient s'appliquer à d'autres email mais la variable contenant le message d'erreur est spécifique à l'attribut "userEmailToInvite". Du coup, j'ai préféré que cela soit claire dans le nom de la fonction de validation.